### PR TITLE
Blog App Rails Validation Models

### PR DESCRIPTION
**Pull request by luccamicale**
## Validation Model Rspec 💯 

##https://github.com/luccamicale/Blog_Rails 👍 

## Checklist before requesting a review 💯 
- [x] I have performed a self-review of my code
- [x] Use validations for models.
- [x] Write unit tests for models. 🥇 🔢 

test

### CODE REVIEW

File: spec/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  it 'posts_id should be present' do
+    subject.post_id = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'author_id should be present' do
+    subject.author_id = nil
+    expect(subject).to_not be_valid
+  end
```
Comment (Deepakdanger) [inline spec/comment_spec.rb:17]: - [x] Kindly include more positive case of validation.

File: spec/like_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  it 'post_id should be present' do
+    subject.post_id = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'author_id should be present' do
+    subject.author_id = nil
+    expect(subject).to_not be_valid
+  end
```
Comment (Deepakdanger) [inline spec/like_spec.rb:17]: - [x] Kindly include more positive case of validation.

File: spec/post_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  it 'author_id should be present' do
+    subject.author_id = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments counter should be integer' do
+    subject.comments_counter = 'String'
+    expect(subject).to_not be_valid
+  end
```
Comment (Deepakdanger) [inline spec/post_spec.rb:21]: - [x] Kindly include more positive case of validation.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @luccamicale , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luccamicale** [comment]
I added more case of validation in my files. 
test

**Comment by luccamicale** [comment]
Before there were 9 and now there are 14

**Comment by luccamicale** [comment]
Now my project should be approved!!!!!!

**Review by fobadara — APPROVED** [review]
Hi @luccamicale, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**